R/createYaml.R

```diff
@@ -0,0 +1,161 @@
+#' Creates a yaml file storing the parameters that are used for creating the PTXQC report, 
+#' optionally returns these parameters and a list of qc-Metrics
+#'
+#' @param yc A yaml class object created by YAMLClass$new()
+#' @param path path to an (empty) yaml file
+#' @param param list of parameters sorted by names (important for shiny application)
+#' @param DEBUG_PTXQC default FALSE
+#' @param get_parameters Should the parameters and qc-Metrics be returned? Default TRUE

```

Comment:
omit this parameter and always return the values.
The user can always choose not to receive them (why would you call the function?)